### DISCUSSION TITLE
Clock and [%] headlines not aligning properly

### DISCUSSION THREAD

**Discussion by rayvermey** [minad/org-modern#214] [Q&A]
Hi

My tags in headlineslines with a CLOCK: (start - end => duration)
and tags in lines where i use [%] for daily goals with - [ ] lines beneath
do not align properly, the CLOCK lines align 8 positions/character before my defined column position
and the Daily Goals header 3 position/characters the defined column.
When i disable org-modern all tags are aligned properly.

Any way around this?

Thanx

Ray

**Comment by minad** [comment]
org-modern relies on variable pitch fonts. With such styling, alignment is not preserved. You can however disable org-modern styling of each syntax element individually, if you prefer to keep the alignment.
